### PR TITLE
Feature/survey redux setup

### LINKED ISSUE #10: Refactor onClickSubscribe method/create new method for StylePreferences redirect

### LINKED ISSUE #20: Survey start page
- [ ] When the user clicks on the Subscribe button on a crate, they're taken to a page with instructions for the survey and a survey start button.
           -Component names:

### LINKED ISSUE #26: Function for navigating forward through survey

### LINKED ISSUE #27: Function for moving backward through survey

### LINKED ISSUE #30: Set up Survey component

### LINKED ISSUE #31: Set up new route for Survey

### PR DESCRIPTION
**Pull request by nickhartdev**
### Type of change made:
- [ ] Bugfix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Testing - building tests
- [ ] Formatting - no new features
- [ ] Styling- no new features
### Detailed Description (Why is this change required? What problem does it solve? Any challenges that arose while implementing this feature? If so, how were the addressed?)
This PR:
- Updates the survey start layout
- Connects the survey component to the store
- Adds methods for survey page navigation

This code was written with everyone present, completed with a driver navigator workflow.
###### Screenshots (if appropriate):
Updated survey start UI

### CODE REVIEW

File: code/web/src/modules/survey/Survey.js [hunk 1]
```diff
@@ -23,16 +24,35 @@ class Survey extends PureComponent {
...
-        </GridCell>
-        <GridCell style={{ textAlign: "center" }}>
-        <Button theme="primary">Click Here!</Button>
-        </GridCell>
-      </Grid>
+      <div>
+        <Grid style={{height: '7em'}}>
+          <GridCell style={{ textAlign: "center", alignCenter: true }}>
+            <H4 style={{'paddingTop': '2em'}}>WELCOME TO THE DRAMARAMA!</H4>
+            <p style={{'marginTop': '1em'}}>Visualize your style.</p>
```
Comment (AurumValian) [inline code/web/src/modules/survey/Survey.js:31]: Visualize your style.

File: code/web/src/modules/survey/api/actions.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    dispatch({
+      type: MOVE_FORWARD
+    })
+  }
+)
+
+export const moveBackward = () => (
+  dispatch => {
+    dispatch({
+      type: MOVE_FORWARD
```
Comment (AurumValian) [inline code/web/src/modules/survey/api/actions.js]: We'll probably want to change this to backward once we hook it up.

**Review by AurumValian — APPROVED** [review]
We've done it! Great job, beans!

**Review by amykarnaze — APPROVED** [review]
We got redux hooked on up! Great thinking switching views with a good ole adding and subtracting. Its so nice having state in store. Very easy to access. Basic styling looks minimal.

File: code/web/src/modules/survey/api/actions.js [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+    })
+  }
+)
+
+export const moveBackward = () => (
+  dispatch => {
+    dispatch({
+      type: MOVE_BACKWARD
```
Comment (nickhartdev) [inline code/web/src/modules/survey/api/actions.js:38]: Just fixed this to have the correct action type!

File: code/web/src/modules/survey/Survey.js (hunk 1 above)
Comment (ejdelsztejn) [reply code/web/src/modules/survey/Survey.js:31]: Love it!

**Review by ejdelsztejn — APPROVED** [review]
Thanks for hopping on that call and talking me through this! Everything looks great so far.

**Pull request merged**